session/bootstrap.go

```diff
@@ -1193,11 +1204,15 @@ func syncUpgradeState(s Session) {
 }
 
 func syncNormalRunning(s Session) {
-	_, err := ddl.ResumeAllJobsBySystem(s)
+	jobErrs, err := ddl.ResumeAllJobsBySystem(s)
 	if err != nil {
 		logutil.BgLogger().Fatal("upgrade pause all jobs failed", zap.Error(err))
 	}
 
+	for _, err = range jobErrs {
+		logutil.BgLogger().Warn("[DDL] unable to resume the job, error: ", zap.Error(err))
```

Comment:
```suggestion
		logutil.BgLogger().Warn("[DDL] upgrade pause all jobs failed, unable to resume the job, error: ", zap.Error(err))
```

Reply:
Could not get the point. The operation here is "Resume", and why the log here is about "Pause".

Reply:
Sorry. It seems that the log of line1209 is wrong. I want we add "upgrade resume all jobs failed" for these logs.